alerter/src/main/java/com/tapadoo/alerter/Alert.java

```diff
@@ -122,6 +123,8 @@ private void initView() {
         inflate(getContext(), R.layout.alerter_alert_view, this);
         setHapticFeedbackEnabled(true);
 
+        ViewCompat.setTranslationZ(this, Integer.MAX_VALUE);
```

Comment:
Seems like overkill to get it to draw above the status bar, at least set it to half of max. Does this cause any issue with shadows or anything? 

Was it always being drawn below it or is this an Oreo issue?

Reply:
Think of translation Z as a layering system, I want to ensure it's drawn over everything. So an arbitrarily large value is the best solution. No issues, as shadows come from the view's elevation value. 

The issue this solves is that when the Alerter is called early on in an Activity's lifecycle, it seems to get drawn under the status bar - In other cases, such as calling it from an on click listener, it functions as expected.